addons/mapMarkers/XEH_postInit.sqf

```diff
@@ -0,0 +1,51 @@
+#include "script_component.hpp"
+
+// Load configurations from the missions config file
+[missionConfigFile] call FUNC(loadFromConfigFile);
+
+// Start drawCache update using the provided updateInterval
+[{[] call FUNC(updateMarkerDrawCache)}, GVAR(updateInterval), []] call CBA_fnc_addPerFrameHandler;
```

Comment:
Do we want to update the cache every interval? Maybe only update the cache on player init event?